docs/admin/authentication.md

```diff
@@ -185,7 +185,7 @@ talk to the API server. Accounts may be explicitly associated with pods using th
 NOTE: `serviceAccountName` is usually omitted because this is done automatically.
 
 ```
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
Looks good - can we add a comment that this is specific to version 1.9? Perhaps something like:

> `apiVersion: apps/v1 # this apiVersion is relevant as of Kubernetes 1.9`

Reply:
Sounds good. Will update.